python/vyos/frrender.py

```diff
@@ -147,6 +148,43 @@ def dict_helper_pim_defaults(pim, path):
         pim = config_dict_merge(default_values, pim)
         return pim
 
+    def dict_helper_nhrp_defaults(nhrp):
+        nflog_redirect = 1
+        nflog_multicast = 2
+
+        nhrp = conf.merge_defaults(nhrp, recursive=True)
+
+        nhrp_tunnel = conf.get_config_dict(['interfaces', 'tunnel'],
+                                           key_mangling=('-', '_'),
+                                           get_first_key=True,
+                                           no_tag_node_value_mangle=True)
+
+        if nhrp_tunnel: nhrp.update({'if_tunnel': nhrp_tunnel})
+
+        for intf, intf_config in nhrp['tunnel'].items():
+            if 'multicast' in intf_config:
+                nhrp['multicast'] = nflog_multicast
+            if 'redirect' in intf_config:
+                nhrp['redirect'] = nflog_redirect
+
+        ##Add ipsec profile config to nhrp configuration to apply encryption
+        profile = conf.get_config_dict(['vpn', 'ipsec', 'profile'],
+                                       key_mangling=('-', '_'),
+                                       get_first_key=True,
+                                       no_tag_node_value_mangle=True)
+
+        for name, profile_conf in profile.items():
+            if 'disable' not in profile_conf:
```

Comment:
You can unindent one entire block by using

```suggestion
            if 'disable' in profile_conf:
                continue
```

Reply:
done